src/class_sqlThread.py

```diff
@@ -461,11 +563,27 @@ def run(self):
                     rowcount = self.cur.rowcount
                 except Exception as err:
                     if str(err) == 'database or disk is full':
-                        logger.fatal('(while cur.execute) Alert: Your disk or data storage volume is full. sqlThread will now exit.')
-                        queues.UISignalQueue.put(('alert', (tr._translate("MainWindow", "Disk full"), tr._translate("MainWindow", 'Alert: Your disk or data storage volume is full. Bitmessage will now exit.'), True)))
+                        logger.fatal(
+                            '(while cur.execute) Alert: Your disk or data storage volume is full.'
+                            ' sqlThread will now exit.')
+                        queues.UISignalQueue.put((
+                            'alert', (
+                                tr._translate("MainWindow", "Disk full"),
+                                tr._translate(
+                                    "MainWindow",
+                                    'Alert: Your disk or data storage volume is full. Bitmessage will now exit.'
+                                ), True)))
                         os._exit(0)
                     else:
-                        logger.fatal('Major error occurred when trying to execute a SQL statement within the sqlThread. Please tell Atheros about this error message or post it in the forum! Error occurred while trying to execute statement: "%s"  Here are the parameters; you might want to censor this data with asterisks (***) as it can contain private information: %s. Here is the actual error message thrown by the sqlThread: %s', str(item), str(repr(parameters)),  str(err))
+                        logger.fatal(
+                            'Major error occurred when trying to execute a SQL statement within the sqlThread. Please'
+                            ' tell Atheros about this error message or post it in the forum! Error occurred while'
+                            ' trying to execute statement: "%s"  Here are the parameters; you might want to censor'
+                            ' this data with asterisks (***) as it can contain private information: %s. Here is the'
+                            ' actual error message thrown by the sqlThread: %s',
+                            str(item),
```

Comment:
`%s` causes `str()` itself - no need to convert args.
use `%r` instead of str(repr(parameters))